ajax/libs/hammer-time/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "hammer-time",
+  "filename": "dist/hammer-time.min.js",
```

Comment:
Please change to `"filename": "hammer-time.min.js",`
we didn't use `dist/` prefix usually.